extensions/blocks/calendly/calendly.php

```diff
@@ -88,41 +89,54 @@ function load_assets( $attr, $content ) {
 
 		// Render deprecated version of Calendly block if needed. New markup block button class before rendering here.
 		if ( false === strpos( $content, 'wp-block-jetpack-button' ) ) {
-			$content = deprecated_render_button_v1( $attr, $block_id, $classes, $url );
+			$content = deprecated_render_button_v1( $attr, $html_id, $classes, $url );
 		} else {
-			$content = str_replace( 'calendly-widget-id', esc_attr( $block_id ), $content );
+			$content = str_replace( 'calendly-widget-id', esc_attr( $html_id ), $content );
 			$content = str_replace( $base_url, $url, $content );
 		}
 
 		if ( ! $is_amp_request ) {
 			wp_add_inline_script(
 				'jetpack-calendly-external-js',
-				sprintf( "calendly_attach_link_events( '%s' )", esc_js( $block_id ) )
+				sprintf( "calendly_attach_link_events( '%s' )", esc_js( $html_id ) )
 			);
 		}
 	} else { // Inline style.
 		if ( $is_amp_request ) {
 			$content = sprintf(
 				'<div class="%1$s" id="%2$s"><a href="%3$s" role="button" target="_blank">%4$s</a></div>',
 				esc_attr( Blocks::classes( FEATURE_NAME, $attr ) ),
-				esc_attr( $block_id ),
+				esc_attr( $html_id ),
 				esc_js( $url ),
 				wp_kses_post( get_attribute( $attr, 'submitButtonText' ) )
 			);
 		} else {
-			$content = sprintf(
+			$content          = sprintf(
 				'<div class="%1$s" id="%2$s"></div>',
 				esc_attr( $classes ),
-				esc_attr( $block_id )
+				esc_attr( $html_id )
 			);
-			$script  = <<<JS_END
-Calendly.initInlineWidget({
-	url: '%s',
-	parentElement: document.getElementById('%s'),
-	inlineStyles: false,
-});
+			$escaped_block_id = esc_js( $block_id );
+			$script           = <<<JS_END
+// Call function with block id in name
+// to allow for multiple widgets.
+function jetpackInitCalendly{$escaped_block_id}() {
```

Comment:
It doesn't look good to me creating a new function for each Calendly instance. I've just tested creating only one `jetpackInitCalendly()` function, and pass the `url` and the `id` attributes as parameters of it, in the javascript context. 
Do you think it worths it? I can push my changes if you agree.

Reply:
> It doesn't look good to me creating a new function for each Calendly instance. I've just tested creating only one `jetpackInitCalendly()` function, and pass the `url` and the `id` attributes as parameters of it, in the javascript context.
> Do you think it worths it? I can push my changes if you agree.

Sure, sounds good! 🙌 The goal was to allow for separate Calendly blocks with separate functions, but if you've found a way to do that with parameters that would be awesome.

Reply:
Handled here 59dd7cd, Allison.
**before**

![image](https://user-images.githubusercontent.com/77539/95344561-27254980-0890-11eb-8f30-e14cb21e50b1.png)


**after**

![image](https://user-images.githubusercontent.com/77539/95343509-04466580-088f-11eb-83fd-e6ffe3a867d5.png)

Reply:
Could we put that `window.jQuery && window.o2` check in the `jetpackInitCalendly` function? It seems like that would get rid of some repeated code.

Reply:
I think it's even nicer.

* Always call the `jetpackInitCalendly()`, passing the proper parameters
* if jQuery & o2 are defined, subscribe to the `o2.ready` event
* If not, just call the init Calendly widget method.

Reply:
> Could we put that `window.jQuery && window.o2` check in the `jetpackInitCalendly` function? It seems like that would get rid of some more repeated code.

> 
> Always call the jetpackInitCalendly(), passing the proper parameters
> if jQuery & o2 are defined, subscribe to the o2.ready event
> If not, just call the init Calendly widget method.

Thanks! 🙌 I made the change. Let me know if you see any ways to streamline it further. 😄

Reply:
I'm afraid the last change breaks the rendering process, Allison. All blicks are rendered at the bottom, in the front-end.

![image](https://user-images.githubusercontent.com/77539/95505735-3e922e80-0985-11eb-83ca-3ea911182cc7.png)